src/streaming/utils/TTMLParser.js

```diff
@@ -725,6 +725,20 @@ function TTMLParser() {
         return 'rgba(' + rgb.join(',') + ',' + alpha + ');';
     }
 
+    // Convert an RGBA value written in TTML rgba(v,v,v,a => 0 to 255) to CSS rgba(v,v,v,a => 0 to 1).
+    function convertAlphaValue(rgbaTTML) {
+        let rgba,
+            alpha;
+        let resu = rgbaTTML;
+
+        rgba = rgbaTTML.replace(/^(rgb|rgba)\(/,'').replace(/\)$/,'').replace(/\s/g,'').split(',');
+        if (rgba[rgba.length - 1] > 1) {
```

Comment:
Why test for alpha value > 1 here ? The range of alpha values in TTML is 0 - 255.